cloud/deploys/azure/bin/db-connection

```diff
@@ -0,0 +1,27 @@
+# example: ./db-connection -g sgdev -d civiform-artistic-halibut -k bastion
```

Comment:
suggestion (blocking): for both scripts

- quote all strings (see https://google.github.io/styleguide/shellguide.html)
- line length limit 80 chars
- add shebang at the top
- add `# DOC: ...` comments along with `...../bin/help` script, can copy from root repo `bin/help`
- validate flag values, at minimum for presence

Reply:
hmmm should these live in that repo since the help script imports some stuff into it

Reply:
okay i think i should have grabbed them all

Reply:
a lil duplicated code here, but i think since the goal is for this to be in a fully different repo it's probably okay-ish

Reply:
I don't think this code will ever be in a fully different repo since it's for our permanent staging server. I'd recommend adding `source bin/lib.sh` at the top, will be less need for duplication then.

Reply:
Even if we do want to switch to dogfooding the deployment system for our staging servers, we want as much as possible to keep shared code in the mainline repo and delegate to it from the forked deployment code. Scripts like this are a good example of the things we want to keep consistent.

This is making me wonder if these bin scripts should be somewhere other than `cloud/deploy/*` since they shouldn't be specific to a single deployment.

Reply:
i mean this one is kind of specific since it includes azure commands?

Reply:
i'd expect they're generic to any azure deployment though?

Reply:
yup! but doesn't that mean they should live in this cloud directory not the bin?

Reply:
yes, you're right it should. i think the thing i'm trying to sort out is how the repo can share shell lib code between the development and deploy scripts so we don't have multiple copies of functions that do basic things like formatting output.

we don't need to figure that out in this PR though.

Reply:
ahh gotcha gotcha

Reply:
made an issue https://github.com/seattle-uat/civiform/issues/1897